packages/core/helper-plugin/src/components/FilterPopoverURLQuery/index.js

```diff
@@ -18,7 +18,8 @@ import getFilterList from './utils/getFilterList';
 const FilterPopoverURLQuery = ({ displayedFilters, isVisible, onBlur, onToggle, source }) => {
   const [{ query }, setQuery] = useQueryParams();
   const { formatMessage } = useIntl();
-  const { trackUsage } = useTracking();
+  // WHY
+  //const { trackUsage } = useTracking();
```

Comment:
@joshuaellis I need your help - not sure if I don't see the problem today or what is going on but I am not able to properly mock `useTracking` even though it is included in the test.

Do you have an idea why?

Reply:
I'm not sure, I can never get mocks to work as intended. IMO we should set `window.strapi.telemetryDisabled` to `true` and wrap everything in the `TrackingContext` then we don't need to mock anything – simpler testing 🥳 

It's on my todo list for the FE testing utils, just not there yet.

Reply:
I did this, so you don't need to mock it anymore fyi